routes/nearbyInterestStack.js

```diff
@@ -0,0 +1,22 @@
+import { createStackNavigator} from "react-navigation-stack";
```

Comment:
<img width="907" alt="Screen Shot 2020-04-01 at 4 25 34 PM" src="https://user-images.githubusercontent.com/35850933/78183146-6d0eda00-7435-11ea-8c5f-11650734a5db.png">

Reply:
Ill change the name of the constant instead

Reply:
sounds good!!